datafusion/datasource/src/file_scan_config.rs

```diff
@@ -706,28 +706,6 @@ impl FileScanConfig {
         }
     }
 
-    /// Set the file source
-    #[deprecated(since = "47.0.0", note = "use FileScanConfigBuilder instead")]
```

Comment:
Technically speaking we should leave these methods until 53.0.0 per https://datafusion.apache.org/contributor-guide/api-health.html#deprecation-guidelines


However, given they are pretty low level (and thus I don't expect many actual users of them) and we are trying to work on this code for other reasons, I think it is fine to remove them early